### PR TITLE
Added labels and reference to the counters file

### PR DESCRIPTION
**Pull request by esbenbach**
Added counter labels for all counters, and added a reference to the EventSource file where the counters are maintained.

The table ends up being rather "ugly" with the long counter names, so I force a line break in the counter name cell. Not sure this is the right approach for controlling the style, let me know if you want if changed.

This should solve #3234

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/EntityFramework.Docs?pullRequest=3526) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/EntityFramework.Docs?pullRequest=3526) before we can accept your contribution.<br/><br/>:x: esbenbach [sign now](https://cla.dotnetfoundation.org/dotnet/EntityFramework.Docs?pullRequest=3526)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/EntityFramework.Docs?pullRequest=3526) it.</sub>

File: entity-framework/core/logging-events-diagnostics/event-counters.md [hunk 1]
```diff
@@ -70,14 +70,16 @@ Press p to pause, r to resume, q to quit.
...
-Queries                               | The number of queries executed.
-Query Cache Hit Rate (%)              | The ratio of query cache hits to misses. The first time a given LINQ query is executed by EF Core (excluding parameters), it must be compiled in what is a relatively heavy process. In a normal application, all queries are reused, and the query cache hit rate should be stable at 100% after an initial warmup period. If this number is less than 100% over time, you may experience degraded perf due to repeated compilations, which could be a result of suboptimal dynamic query generation.
-SaveChanges                           | The number of times `SaveChanges` has been called. Note that `SaveChanges` saves multiple changes in a single batch, so this doesn't necessarily represent each individual update done on a single entity.
+The following counters are available and maintained in [`EntityFrameworkEventSource.cs`](https://github.com/dotnet/efcore/blob/main/src/EFCore/Infrastructure/EntityFrameworkEventSource.cs)
```
Comment (smitpatel) [inline entity-framework/core/logging-events-diagnostics/event-counters.md:73]: Cannot add link to source here.

Comment (esbenbach) [reply entity-framework/core/logging-events-diagnostics/event-counters.md:73]: Okay, ill remove it. Just figured I would keep it similar to this https://docs.microsoft.com/en-us/dotnet/core/diagnostics/available-counters

**Comment by esbenbach** [comment]
Any chance this is going to get approved? Otherwise I can just abandon it.

**Comment by roji** [comment]
@esbenbach sorry this didn't receive any attention - it came just as the 6.0 release was being prepared. I'll review this in the coming days.

**Review by roji — APPROVED** [review]

**Pull request merged**